### PR TITLE
Update spectre theme to 1.0.2

### PR DESCRIPTION
**Pull request by lnlion**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -647,6 +647,18 @@
         "remote_install": true,
         "compatible_version": ">=1.0.37"
     },
+    "spectre": {
+        "title": "Spectre Theme",
+        "version": "1.0.2",
+        "author": "Viggo",
+        "license": "CC-BY-4.0",
+        "description": "omizer plugin.",
+        "homepage": "Modern And Simple CSS Theme For Kanboard",
```
Comment (fguillot) [inline plugins.json]: typo in homepage and description fields?

Comment (kuerme) [reply plugins.json]: fixed

Comment (lnlion) [reply plugins.json]: > typo in homepage and description fields?

fixed

File: plugins.json [hunk 2]
```diff
@@ -647,6 +647,18 @@
...
     },
+    "spectre": {
+        "title": "Spectre Theme",
+        "version": "1.0.2",
+        "author": "Viggo",
+        "license": "CC-BY-4.0",
+        "description": "Modern And Simple CSS Theme For Kanboard",
+        "homepage": "https://github.com/kuerme/kanboard-theme-spectre",
+        "readme": "https://raw.githubusercontent.com/kuerme/kanboard-theme-spectre/master/README.md",
+        "download": "https://github.com/kuerme/kanboard-theme-spectre/releases/download/v1.0.2/Spectre.zip",
```
Comment (fguillot) [inline plugins.json]: The download URL returns a 404.

It should be `https://github.com/kuerme/kanboard-theme-spectre/releases/download/v1.0.2/Spectre-1.0.2.zip`

**Review by fguillot — CHANGES_REQUESTED** [review]

**Pull request merged**